infra/charts/turing/.helmignore

```diff
@@ -0,0 +1 @@
+../.helmignore
```

Comment:
What does this do? Does this ignore the patterns defined inside `../.helmignore` ?

Reply:
Yup, works similarly to `.gitignore` and `.dockerignore` – it excludes provided paths from being packaged with the chart

Reply:
Oh my question was actually about why the content of this file is `../.helmignore`. But I realise now that it's actually symlinked to `../.helmignore`. Makes sense.